navigation/ca_move_base/launch/navigate.launch

```diff
@@ -11,12 +11,12 @@
   </include>
 
 	<!-- AMCL -->
-  <include if="$(eval arg('slam') == False)" file="$(find ca_move_base)/launch/amcl.launch">
+  <include if="$(eval str(arg('navigation')) == 'amcl')" file="$(find ca_move_base)/launch/amcl.launch">
     <arg name="ns" value="$(arg robot_name)"/>
   </include>
 
   <!-- slam_gmapping -->
-  <include if="$(eval arg('slam') == True)" file="$(find ca_slam)/launch/slam_gmapping.launch">
+  <include if="$(eval str(arg('navigation')) == 'slam')" file="$(find ca_slam)/launch/slam_gmapping.launch">
```

Comment:
Please add a case when the variable is `none` (neither amcl or slam) so no localization is loaded, in case we don't need it.